presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java

```diff
@@ -327,4 +362,18 @@ private static SqlVarbinary sqlVarbinaryHex(String value)
     {
         return new SqlVarbinary(base16().decode(value));
     }
+
+    private static SqlVarbinary varbinary(String string)
+    {
+        return new SqlVarbinary(string.getBytes());
+    }
+
+    private static SqlVarbinary varbinary(int... bytesAsInts)
```

Comment:
I can't find a way to convert an int array to a byte array using streams. Let's see how other people comment on this.

Reply:
This is the same implementation as [Line#642-649](https://github.com/prestodb/presto/blob/master/presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java#L642~L649) in TestStringFunctions.java.